docs/src/main/asciidoc/command-mode-reference.adoc

```diff
@@ -195,12 +207,15 @@ public class HelloTest {
     public void testManualLaunch(QuarkusMainLauncher launcher) {
         LaunchResult result = launcher.launch("Everyone");
         Assertions.assertEquals(0, result.exitCode());
-        Assertions.assertEquals("Hello Everyone", result.getOutput());
+        Assertions.assertTrue(result.getOutput().contains("Hello Everyone"));
     }
 }
 
 ----
 
+Each test method must be annotated by `@Launch` to automatically start the application of have a `QuarkusMainLauncher`
```

Comment:
```suggestion
Each test method must be annotated with `@Launch` to automatically start the application of have a `QuarkusMainLauncher`
```

and there's something off with `start the application of have a` but I don't know what you wanted to say.

Reply:
It's not `of` but `or` and apparently the end of the sentence is missing.